js/bundle/package.json

```diff
@@ -9,6 +9,9 @@
     "build": "tsc",
     "test": "jasmine tests/*.js"
   },
+  "bin": {
+    "gen-webbundle": "./bin/gen-webbundle"
```

Comment:
IMO: Just using package name (wbn) as a command name does more make sense for me. Why do you make this name?

Reply:
Yeah, that makes sense to me. Renamed the command.

We may add other CLI features in the future (such as `dump-bundle` equivalent), but then we could switch to the subcommand style (e.g. `wbn gen ...` / `wbn dump ...`).